pkg/cmd/server/api/types.go

```diff
@@ -47,10 +47,14 @@ type MasterConfig struct {
 	// CORSAllowedOrigins
 	CORSAllowedOrigins []string
 
+	// MasterPublicURL is how clients can access the OpenShift API server
+	MasterPublicURL string
```

Comment:
We don't validate well formedness of the others.

Also, why do they have to match?

> On Apr 17, 2015, at 8:12 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/cmd/server/api/types.go:
> 
> > @@ -47,10 +47,14 @@ type MasterConfig struct {
> >  	// CORSAllowedOrigins
> >  	CORSAllowedOrigins []string
> >  
> > +	// MasterPublicURL is how clients can access the OpenShift API server
> > +	MasterPublicURL string
> Add validation to ensure the URL is present (required for swagger), well-formed, and matches any other MasterPublicURLs that may be present.
> 
> —
> Reply to this email directly or view it on GitHub.
>